index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">   
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+        <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+        <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+
+
+    </head>
+
+    <body>
+
+        <header class="navbar container">
+            <a href="https://github.com/qusai122" id="logo">
+                Qusai Issa
+            </a>
+            <nav>
+                <div class="menu">
+                    <i class="fa-solid fa-bars"></i>
+                </div>
+            </nav>
+        </header>
+        <section class="section2">
+            <div class="container">
+                <h1 class="welcome">
+                    I’m Qusai, <br/> Glad to see you!
+                </h1>
+                <p class="about_me">
+                    I’m a full-stack software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <h3 class="connect_me">
+                    <a href="https://github.com/qusai122">LET'S CONNECT</a>
+                </h3>
+                <ul class="icon-group">
+                    <li class="icon"> <a href="https://github.com/qusai122"><i class="fa-brands fa-github"></i></a></li>
+                    <li class="icon"> <a href="https://www.linkedin.com/in/qusai-issa-ab5a59156/"><i class="fa-brands fa-linkedin"></i></a></li>
+                    <!--li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li-->
+                    <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li class="icon"> <a href="mailto:qusai.b.issa@gmail.com"><i class="fa-solid fa-m"></i></a></li>
+                </ul>
```

Comment:
- [ ] [OPTIONAL] As observed in the Figma design, there are five icons, but you have only four here. Also, the color of these icons is not the same as what we have in the Figma design. Kindly fix.

![image](https://user-images.githubusercontent.com/84156714/219610071-b09dc7d3-e789-4a73-b44c-9aef531c18bd.png)

**_Note_** - this is marked as **_Optional_** because the previous reviewer did not mention it. Kindly fix this as it is a major observable issue and requirement.

Reply:
he didn't mention it because it was there
I removed it because I didn't use
but its in again.✔